internal/version/version.go

```diff
@@ -248,3 +250,27 @@ func (i *Info) JSONString() (string, error) {
 	}
 	return string(b), nil
 }
+
+// GetLatestReleaseVersion fetches the latest github release
+// version from cri-o.
+func GetLatestReleaseVersion(token string) (string, error) {
```

Comment:
this seems like something we want in scripts/release-notes instead. otherwise, we're suddenly linking the github dependencies in cri-o for something we're only using as a cri-o utility